compiler/testdata/errors.go

```diff
@@ -16,14 +16,6 @@ type Uint uint32
 //go:wasmimport modulename validparam
 func validparam(a int32, b uint64, c float64, d unsafe.Pointer, e Uint)
```

Comment:
Might be a good idea to update this to add some new types (like `string` and `*struct{}`).

Reply:
Fixed in https://github.com/dgryski/tinygo/pull/17